src/widgets/_toolbars.scss

```diff
@@ -2,3 +2,15 @@ actionbar,
 .search-bar {
     background-color: bg-color(3);
 }
+
+toolbar separator {
```

Comment:
Do you have an example of where this is used?

Reply:
Some apps use this to make "dividers" in their toolbars.